### PR TITLE
chore(Typography): document typography props

### PR DESCRIPTION
**Pull request by joakbjerk**
## Summary

Documenting the props for P and H elements

### CODE REVIEW

File: packages/dnb-eufemia/src/elements/typography/P.tsx [hunk 1]
```diff
@@ -20,11 +20,32 @@ export type PSize =
...
+    /**
+     * Tells the component to use the small font-size dnb-p__size--small defined in paragraphStyle - typography-mixins.scss. Find more details here https://eufemia.dnb.no/uilib/typography/font-size/
+     */
     small?: boolean
+    /**
+     * Tells the component to use the medium font-weight styling dnb-p--medium defined in paragraphStyle - typography-mixins.scss. Find more details here https://eufemia.dnb.no/uilib/typography/font-weight/
+     */
     medium?: boolean
+    /**
+     * Tells the component to use the medium font-weight styling dnb-p--bold defined in paragraphStyle - typography-mixins.scss. Find more details here https://eufemia.dnb.no/uilib/typography/font-weight/
```
Comment (langz) [inline packages/dnb-eufemia/src/elements/typography/P.tsx:37]:
```suggestion
     * Tells the component to use the bold font-weight styling dnb-p--bold defined in paragraphStyle - typography-mixins.scss. Find more details here https://eufemia.dnb.no/uilib/typography/font-weight/
```

File: packages/dnb-eufemia/src/elements/typography/P.tsx [hunk 2]
```diff
@@ -20,11 +20,32 @@ export type PSize =
...
+     */
     bold?: boolean
+    /**
+     * Sets the font size based on size classes defined in paragraphStyle - typography-mixins.scss. For more detailed information go here: https://eufemia.dnb.no/uilib/typography/font-size/
+     */
     size?: PSize
+    /**
+     * A string containing a combination of modifiers, used to set both font-size and weight in one property. e.g. "x-small bold" would make the paragraph extra small and bold.
+     * Works as a felxible alternativ to setting the medium, small, bold and size props.
```
Comment (langz) [inline packages/dnb-eufemia/src/elements/typography/P.tsx:46]:
```suggestion
     * Works as a flexible alternative to setting the medium, small, bold and size props.
```

File: packages/dnb-eufemia/src/elements/typography/P.tsx [hunk 3]
```diff
@@ -20,11 +20,32 @@ export type PSize =
...
+     * List of modifieres can be found at https://eufemia.dnb.no/uilib/typography/font-size/ and https://eufemia.dnb.no/uilib/typography/font-weight/
```
Comment (langz) [inline packages/dnb-eufemia/src/elements/typography/P.tsx:47]:
```suggestion
     * List of modifiers can be found at https://eufemia.dnb.no/uilib/typography/font-size/ and https://eufemia.dnb.no/uilib/typography/font-weight/
```

File: packages/dnb-eufemia/src/elements/typography/P.tsx [hunk 4]
```diff
@@ -20,11 +20,32 @@ export type PSize =
...
   React.HTMLAttributes<HTMLParagraphElement> & {
+    /**
+     * Defines the Element Type, like "p"
+     * Default: p
+     */
     element?: DynamicElement & 'p'
+    /**
+     * Tells the component to use the small font-size dnb-p__size--small defined in paragraphStyle - typography-mixins.scss. Find more details here https://eufemia.dnb.no/uilib/typography/font-size/
+     */
     small?: boolean
```
Comment (langz) [inline packages/dnb-eufemia/src/elements/typography/P.tsx:31]: Nothing wrong with the documentation part here, but isn't it a bit strange that we have `small` as property? 🤔 
Couldn't we just use `size="small"`?

**Review by langz — APPROVED** [review]

File: packages/dnb-eufemia/src/elements/typography/P.tsx (hunk 4 above)
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/typography/P.tsx:31]: Yeah I agree, unless there is something I don't understand here about the `small` prop. We already offer the developers flexibility of setting size with both the `size` and `modifier` props. @tujoworker  what do you think? 🤔

Comment (tujoworker) [reply packages/dnb-eufemia/src/elements/typography/P.tsx:31]: Good point! Let's remove it and note it in the v10 docs. 

@joakbjerk Maybe in another PR. I'll add a task.

**Pull request merged**